File: BasicSampleJava/app/src/main/java/com/regula/facesample/util/LivenessResponseUtil.java
package com.regula.facesample.util;

import android.content.Context;
import android.widget.Toast;

import com.regula.facesdk.model.results.LivenessResponse;

public class LivenessResponseUtil {

   public static void response(Context context, LivenessResponse livenessResponse) {
      if (livenessResponse.getException() != null)
         Toast.makeText(context, "Error: " + livenessResponse.getException().getMessage(), Toast.LENGTH_SHORT).show();
      else
         Toast.makeText(context, "Liveness status: " + livenessResponse.getLiveness().name(), Toast.LENGTH_SHORT).show();
   }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/util/LivenessResponseUtil.kt
package com.regula.facesamplekotlin.util

import android.content.Context
import android.widget.Toast
import com.regula.facesdk.model.results.LivenessResponse

object LivenessResponseUtil {
    fun response(context: Context?, livenessResponse: LivenessResponse) {
        if (livenessResponse.exception != null)
            Toast.makeText(context, "Error: " + livenessResponse.exception!!.message, Toast.LENGTH_SHORT).show()
        else
            Toast.makeText(context, "Liveness status: " + livenessResponse.liveness.name, Toast.LENGTH_SHORT).show()
    }
}